cypress/support/commands/p2p.js

```diff
@@ -208,6 +244,16 @@ Cypress.Commands.add('c_removeExistingAds', () => {
   cy.findByRole('button', { name: 'Delete' }).should('not.exist', {
     timeout: 10000,
   })
```

Comment:
can be used with above line 243 like
```suggestion
  cy.findByRole('button', { name: 'Delete' }).should('be.enabled').click().should('not.exist', {
    timeout: 10000,
  })
```